blueprints.yaml

```
@@ -39,6 +39,28 @@ form:
       accept:
         - image/*
 
+    use_custom_logo_alt:
+      type: toggle
+      label: Use Alternate Logo For Scrolling
+      description: Toggle to use an alternate logo in the navigation menu when the site is scrolled
+      highlight: 0
+      default: 0
+      options:
+        1: PLUGIN_ADMIN.YES
+        0: PLUGIN_ADMIN.NO
+      validate:
+        type: bool
+    
+    custom_logo_alt:
```

I think this naming will be a bit more intuitive.

```suggestion
    custom_logo_scrolled:
```